src/WebJobs.Script/Workers/Rpc/Configuration/WorkerConfigurationResolverOptions.cs

```diff
@@ -9,6 +11,52 @@ namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc.Configuration
 {
     public sealed class WorkerConfigurationResolverOptions : IOptionsFormatter
     {
+        /// <summary>
+        /// Gets or sets the value of platform release channel.
+        /// </summary>
+        public string ReleaseChannel { get; set; }
+
+        /// <summary>
+        /// Gets or sets the value of worker runtime.
+        /// </summary>
+        public string WorkerRuntime { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether placeholder mode is enabled.
+        /// </summary>
+        public bool IsPlaceholderModeEnabled { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether it is a multi-language worker environment.
+        /// </summary>
+        public bool IsMultiLanguageWorkerEnvironment { get; set; }
+
+        /// <summary>
+        /// Gets or sets the list of probing paths for worker resolution.
+        /// </summary>
+        public List<string> ProbingPaths { get; set; }
+
+        /// <summary>
+        /// Gets or sets the worker runtimes available for resolution via Hosting configuration.
+        /// </summary>
+        public HashSet<string> WorkersAvailableForResolution { get; set; }
+
+        /// <summary>
+        /// Gets or sets the dictionary containing language workers related settings in configuration.
+        /// </summary>
+        public Dictionary<string, string> LanguageWorkersSettings { get; set; }
```

Comment:
i mentioned it elsewhere -- but i think this needs to be `IDictionary<string, RpcWorkerDescription>>`. This type confused me and we shouldn't require the value to be parsed later.